openverse_catalog/dags/common/popularity/sql.py

```diff
@@ -306,7 +306,7 @@ def create_audioset_view_query():
     """
     return dedent(
         f"""
-        CREATE VIEW public.{AUDIOSET_VIEW_NAME}
+        CREATE VIEW public.{AUDIO_VIEW_NAME}
```

Comment:
Oh no sorry, I think _this_ one was correct as `AUDIOSET_VIEW_NAME`

Reply:
Oh god, okay let me get it right this time 😅